tart_of_protocol_header + i] != ' '; i++)

```diff
@@ -0,0 +1,2243 @@
+WireKrill 0.0.1 Network Analyser.
```

Comment:
What is this file for? Did you accidentally type some funny command into VI that caused it to save? It looks like it might be example output of the program running ?